helpers/github.ts

```diff
@@ -377,6 +377,14 @@ export async function createDevPoolIssue(projectIssue: GitHubIssue, projectUrl:
   // if issue is "closed" then skip it, no need to copy/paste already "closed" issues
   if (projectIssue.state === "closed") return;
 
+  // if reposistory is archived skip it 
+  if (!projectIssue.repository || projectIssue.repository.archived === true) {
+    if (!projectIssue.repository) {
```

Comment:
Checking the same condition twice doesn't make sense you need to refactor your logic